cmd/slinky-config/main.go

```diff
@@ -84,8 +84,8 @@ var (
 		// -----------------------------------------------------------	//
 		// ---------------------Start API Providers--------------------	//
 		// -----------------------------------------------------------	//
-		binance.Name:       binance.DefaultNonUSMarketConfig,
-		coinbaseapi.Name:   coinbaseapi.DefaultMarketConfig,
+		binance.Name:       binance.DefaultNonUSProviderConfig,
+		coinbaseapi.Name:   coinbaseapi.DefaultProviderConfig,
```

Comment:
Why is this renamed to `DefaultProviderConfig` instead of `DefaultMarketConfig`?

Reply:
They are all going to be renamed provider config - but this must have accidentally been included

https://github.com/skip-mev/slinky/pull/296